website/docs/guides/working-with-guides/core/1-setting-up-examples-on-core-recommended.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Setting up example projects in GitHub Codespaces"
+id: 1-setting-up-examples-on-core-recommended
+slug: working-with-guides/core/1-setting-up-recommended
+description: Learn how to set up Guides example projects on dbt Core in GitHub Codespaces.
+displayText: dbt Core recommended setup
+hoverSnippet: Learn how to set up Guides example projects on dbt Core in GitHub Codespaces.
+---
+
+## Getting started
+
+:::tip Checkout the README
+These instructions along with animated gifs are provided as a walkthrough directly on the repo you'll be using. Feel free to follow the instructions there!
+:::
+
+1. 🔀 **Fork the repo**. Go to the repo specificed as the example project in the guide you're working through, or start from [the base template](https://github.com/dbt-labs/jaffle-shop-template). Click `Use this template` and `Create new repository`.
+2. 🚀 **Open the codespace**. Navigate to that new repository. Where you would normally clone it from the Code button, instead go to the Codespaces tab and click `Create codespace on main`.
```

Comment:
```suggestion
2. 🚀 **Open the codespace**: To navigate to the new repository, go to the Codespaces tab and click **Create codespace on main** instead of cloning it from the Code button as usual.
```